test/examples/hello_world.compose.yml

```diff
@@ -11,33 +11,32 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-version: "3"
```

Comment:
The version has been removed from all compose files since it throws a warning of deprecatation and can create confusion